crypto/asn1/a_gentm.c

```diff
@@ -157,10 +157,10 @@ int ASN1_GENERALIZEDTIME_print(BIO *bp, const ASN1_GENERALIZEDTIME *tm)
     if (BIO_printf(bp, "%s %2d %02d:%02d:%02d%.*s %d%s",
                    _asn1_mon[M - 1], d, h, m, s, f_len, f, y,
                    (gmt) ? " GMT" : "") <= 0)
-        return (0);
+        return 0;
     else
-        return (1);
+        return 1;
  err:
     BIO_write(bp, "Bad time value", 14);
```

Comment:
I don't see any problem with this... since the caller couldn't get the data he/she wants, it should go to error handling